workspaces/sandbox/plugins/sandbox/src/components/SignIn/RHSSOSignInPage.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import { SignInPage } from '@backstage/core-components';
+import { keycloakApiRef } from '../../api';
+
+export const RHSSOSignInPage = (props: any) => {
+  return (
+    <div style={{ visibility: 'hidden' }}>
+      <SignInPage
+        {...props}
+        auto
+        provider={{
+          id: 'oidc',
+          title: 'Red Hat SSO',
+          message: 'Sign in using your Red Hat SSO account',
```

Comment:
```suggestion
          message: 'Sign in using your Red Hat account',
```

nit: "Red Hat SSO" could be confused by Red Hatters as the employee internal SSO. In the Red Hat login page, these accounts are referred to as "Red Hat accounts", so I think it'd be better call them that